modules/utils/getNpmPackageInfo.js

```diff
@@ -7,6 +7,7 @@ function cleanPackageConfig(packageConfig) {
   return {
     name: packageConfig.name,
     version: packageConfig.version,
+    main:  packageConfig.unpkg || packageConfig.main,
```

Comment:
Good idea! (if this is what caused the issue, 😄 I haven't tested either!)

there's already quite a bit of logic in [filenameRedirect](https://github.com/unpkg/unpkg.com/blob/19b2e5574b5ab5cb711ddbf27c5300d114bed4dc/modules/middleware/fetchPackage.js#L51-L89) about these things. I think it might be better to directly map `packageConfig.main`, `packageConfig.module`, `packageConfig.browser`, `packageConfig.unpkg` to this object